Forms/Form.php

```diff
@@ -1377,14 +1370,17 @@ public function getLegend() {
 	}
 
 	/**
-	 * Returns last result of validation, or null if not validated.
-	 * Note: Does not persist between requests in session.
-	 * @todo Maybe it should?
+	 * Returns boolean if there are any errors set for this form, or null if not validated.
 	 *
 	 * @return bool|null
 	 */
 	public function isValid() {
```

Comment:
That's pretty misleading, since `isValid()` relies on `setupFormErrors()` to be called beforehand. Can we avoid creating a new `Form->isValid()` API for this and simply use `Form->validate()`?